src/components/section/completed-section/index.tsx

```diff
@@ -0,0 +1,17 @@
+import classNames from './completed-section.module.scss';
+import Complete from '../../challenges/complete';
+
+const CompletedSection = (props) => {
+  let completeContent;
+
+  if (props.screen == 'Challenges') {
```

Comment:
Why's this component checking what screen it's in? Should it not be the screen's responsibility to render the correct component?